### PR TITLE
Add HIP4082

### PR DESCRIPTION
**Pull request by L-Warlok**
https://www.renesas.com/www/doc/datasheet/hip4082.pdf
HIP4082

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] Provide a screenshot of the symbol(s) from the symbol editor with the pin types visible
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

---

Be patient, we maintainers are volunteers with limited time and need to check your contribution against the datasheet. You can speed up the process by providing all the necessary information (see above). And you can speed up the process even more by providing additional info like the screenshot of the symbol editor pin table (or for high pin counts converted to csv) sorted in the same way as the pin table in the datasheet and a direct link to the datasheet page that contains the pin table.

### CODE REVIEW

**Comment by evanshultz** [comment]
@L-Warlok 
Welcome and thanks for contributing!

1. For new contributions, we want to see fully-specified symbols. So please pick one package for this symbol, or split it into two symbols where each symbols is assigned one package. The symbol name(s) will need to be updated to capture the package. You can find more information at 
http://kicad-pcb.org/libraries/klc/.
1. Put the package at the end of the description.
1. Wildcard the temp rating.
1. I'm not sure it's required to make this symbol interchangeable with HIP4081, so it can be shrunk vertically. Are you thinking that keeping the common pins in the same place is useful?

**Comment by L-Warlok** [comment]
@evanshultz 
Thanks for responding to my pull request :) 

I have duplicated the symbol and put the package at the end of the name. I also shrunk the symbols vertically, I had initially just copied the HIP4081 symbol so that's why it was longer. 

I'm not sure what you mean about the temp rating. Is there a section in the KLC about this?

File: Driver_FET.dcm [hunk 1]
```diff
@@ -163,7 +163,7 @@ F http://www.intersil.com/content/dam/Intersil/documents/hip4/hip4080a.pdf
 $ENDCMP
 #
 $CMP HIP4082
-D 80V, 1.25A Peak Current H-Bridge FET Driver
+D 80V, 1.25A Peak Current H-Bridge FET Driver SOIC Package
```
Comment (evanshultz) [inline Driver_FET.dcm:172]: Change ` SOIC package` to `, SOIC-16`

**Comment by evanshultz** [comment]
Yes. See https://kicad-pcb.org/libraries/klc/S2.2/.

I have also noted some other fixes.

File: Driver_FET.lib [hunk 2]
```diff
@@ -701,6 +701,68 @@ X LDEL 9 -500 -600 100 R 50 50 1 1 P
...
 #
+# HIP4082_DIP
+#
+DEF HIP4082_DIP U 0 20 Y Y 1 F N
+F0 "U" 100 650 50 H V L CNN
+F1 "HIP4082_DIP" 100 550 50 H V L CNN
+F2 "Package_DIP:DIP-16_W7.62mm" 100 750 50 H I L CNN
+F3 "" 100 750 50 H I C CNN
+$FPLIST
+ DIP-*_W7.62mm*
```
Comment (evanshultz) [inline Driver_FET.lib:712]: No dashes or underscores. Underscores are to replaced with an asterisk and dashes are removed.

File: Driver_FET.dcm [hunk 3]
```diff
@@ -162,6 +162,18 @@ K Half Bridge Gate Driver
 F http://www.intersil.com/content/dam/Intersil/documents/hip4/hip4080a.pdf
 $ENDCMP
 #
+$CMP HIP4082_DIP
```
Comment (evanshultz) [inline Driver_FET.dcm:165]: Wildcarding the temp in the MPN will change the `_DIP` to `xB` and you can leave off the trailing `Z`.

**Comment by evanshultz** [comment]
Thanks! This is looking great! Please take a closer look at the first comment inline: add a comma before the package info in the description and remove the final ` Package`. Then it's ready to merge.

**Comment by evanshultz** [comment]
Thanks so much!

**Pull request merged**